### PR TITLE
Fix clue starved strike efficiency

### PR DESCRIPTION
**Pull request by padiwik**

### CODE REVIEW

File: client/src/game/ui/action.ts [hunk 1]
```diff
@@ -508,7 +508,8 @@ actionFunctions.set('strike', (data: ActionStrike) => {
   const strikeX = globals.elements.strikeXs[i];
 
   // Update the stats
-  globals.cluesSpentPlusStrikes += 1;
+  // In clue starved variants, each strike only "costs" half a clue
+  globals.cluesSpentPlusStrikes += globals.variant.name.startsWith('Clue Starved') ? 0.5 : 1;
```
Comment (robertos) [inline client/src/game/ui/action.ts:512]: variantRules.isClueStaved(variant)
Let's not add more string based comparisons now that there is a better way

**Comment by robertos** [comment]
Fix #1219 
Let's try to comment on the related issues so that 1. No need to close manually and 2. The issue opener or other people can easily check the fix

File: client/src/game/ui/action.ts (hunk 1 above)
Comment (padiwik) [reply client/src/game/ui/action.ts:512]: is that implemented already? i would expect the rest of the code to be refactored to use this

Comment (Zamiell) [reply client/src/game/ui/action.ts:512]: variants.ts line 7

Comment (robertos) [reply client/src/game/ui/action.ts:512]: I'll log an issue to replace this kind of thing everywhere

**Pull request merged**